app/main.py

```diff
@@ -73,6 +73,8 @@ async def knowledge_panel(
     facet_tag are the list of values connecting to FacetName
     eg:- category/beer, here beer is the value
     """
+    if facet_tag.country and value_tag is None:
+        raise HTTPException(status_code=400, detail="Value_tag can't be empty!")
```

Comment:
Why this ?

Reply:
To check the edge cases currently if facet_tag is country and value _tag is none then it returns "interval sever error" so to avoid this I just add this

Reply:
Ok, but I think it's better to handle that more deep into the code, and avoid sending back error altogether.

I propose we remove those line of codes for now, and treat it in another PR.

Reply:
Great!